README.md

```diff
@@ -40,16 +40,93 @@ The Rucio file catalog is much flatter than that of a usual `POSIX` filesystems
 - Fix Mac OS X mouting issues
 - Surely a lot more...
 
-## How to build
```

Comment:
General: since for MD it makes no difference, go to new line after every phrase and avoid having periods in the same line, for better gitability :D

Reply:
Yeah, you're right! It provides better readability for uncompiled MD files.